common/app/views/fragments/header.scala.html

```diff
@@ -75,12 +75,11 @@
                         }
 
                         @if(editionId == "uk") {
-                            <a class="top-bar__item hide-until-tablet"
+                            <a class="top-bar__item hide-until-desktop"
```

Comment:
What is the reason for this change / has it been checked with the relevant people?

Reply:
The change doesn't affect what users see. The word dating was hidden until desktop but oddly it's parent was hidden until tablet, so I just removed the nested hides, to prevent an empty parent sitting there.